drivers/i2c/i2c_mchp_xec.c

```diff
@@ -457,7 +457,7 @@ static int i2c_xec_transfer(const struct device *dev, struct i2c_msg *msgs,
 	return 0;
 }
 
-static void i2c_xec_bus_isr(void *arg)
+static void i2c_xec_bus_isr(const struct device *arg)
 {
 #ifdef CONFIG_I2C_SLAVE
 	struct device *dev = (struct device *)arg;
```

Comment:
This should be cleaned up, imho.

Reply:
the script  could not match this case. I'll update it.

Reply:
looks like cocci does not like the #ifdef. it should be part of the '...' statement, but no.